packages/@react-stately/color/src/Color.ts

```diff
@@ -123,6 +128,10 @@ class RGBColor implements ColorValue {
     }
   }
 
+  toInt() {
+    return this.red * Math.pow(16, 4) + this.green * Math.pow(16, 2) + this.blue;
```

Comment:
Math.pow(16, 4) is a constant, same with 16, 2 lets only calculate them once when the module loads